### PR TITLE
Update the information

### PR DESCRIPTION
**Pull request by MrSeccubus**
Update the information. 
Websites are (mostly) not hosted on GitHub, just the source code.
Highlighted to repos.

Added a brief blurp about who we are.

### CODE REVIEW

**Comment by MrSeccubus** [comment]
This should also address #1 , without describing all public repos.

File: README.md
Comment (incyi) [inline README.md:1]: THis. Extra capital. Typo on line 3

Comment (MrSeccubus) [reply README.md:1]: Typo fixed.

**Review by incyi — APPROVED** [review]
For me it looks fine.

**Pull request merged**